index.html

```diff
@@ -289,5 +289,78 @@ <h2>Skills</h2>
         </li>
       </ul>
     </section>
+
```

Comment:
- [ ] **[OPTIONAL]** Great work on the contact section of your portfolio! **It's looking and functioning really well**. I just wanted to **offer a small, optional suggestion for your `about me` section**. 🙋‍♂️ I noticed that the `background image` on the right side **is pushing outside of the designated area**. This could be due to the use of **absolute positioning and too much space on the right**. To make it look more polished and error-free, I recommend **decreasing the amount of space on the right**. I'll leave a video below to show you exactly where the issue is. 🎯

  [10_2023_10-20-25 AM.webm](https://github.com/Arslan49sp/PortfolioMicro/assets/107952724/56f22c17-2c32-4141-8e3d-0317316971ce)

  #### HINT:
  https://github.com/Arslan49sp/PortfolioMicro/blob/88fb19bbae75830f7f7b79e9279e42419f74a554/styles.css#L288-L296